recipes/dbus/dbus-glib.yaml

```diff
@@ -0,0 +1,54 @@
+inherit: [autotools, autoconf]
```

Comment:
Drop `autoconf` class? There is no `checkoutScript` calling `autoconfReconfigure`...